test/e2e/pivoting_based_feature_test.go

```diff
@@ -36,6 +36,7 @@ var _ = Describe("Testing features in ephemeral or target cluster", func() {
 	})
 
 	It("Should get a management cluster then test cert rotation and node reuse", func() {
+		Expect(true).To(BeFalse())
```

Comment:
For a quick test you can also add this to all the feature tests because the order is random and we do not know which will go first

Reply:
is it okay now? I have added it to all the feature tests.